src/main/java/commands/FindCommand.java

```diff
@@ -0,0 +1,26 @@
+package commands;
+
+import main.TaskList;
+import storage.Storage;
+import ui.Ui;
+import tasks.Task;
+
+public class FindCommand extends Command {
```

Comment:
Perhaps you forgot to include javadocs for this? But overall, nice job abstracting into different command classes! :)